apps/src/turtle/turtle.js

```diff
@@ -103,7 +103,7 @@ var Artist = function () {
 
   // image icons and image paths for the 'set pattern block'
   this.lineStylePatternOptions = [];
-  this.stamps = [];
+  this.stckers = [];
```

Comment:
Ugh, nice catch. We set this.stickers on line 174 so I'm just going to delete this line.